src/models.py

```diff
@@ -23,21 +23,18 @@ class User(Base):
     def _get_password_hash(self, password: str) -> str:  # pylint: disable=no-self-use
         return hashlib.sha256(password.encode()).hexdigest()
 
-    def set_password(self, password: str) -> None:
-        """Use this method to set hashed password to User."""
-        self.password_hash = self._get_password_hash(password)
-
     def check_password(self, password_to_check: str) -> bool:
         """Use this method to check User's password."""
         return self._get_password_hash(password_to_check) == self.password_hash
 
     @classmethod
-    def create_user(cls, username: str, password: str) -> None:
+    def create_user(cls, username: str, password: str) -> User:
         """Use this method to create a new user."""
-        new_user = cls(username=username, password_hash=hashlib.sha256(password.encode()).hexdigest())
+        new_user = cls(username=username, password_hash=cls()._get_password_hash(password))
```

Comment:
If we turn `_get_password_hash` into a class method, then we will be able to call it like `User._get_password_hash()` and don't create a redundant instance `cls()`